runtime/strategies/group-view-connections.js

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) 2017 Google Inc. All rights reserved.
+// This code may only be used under the BSD style license found at
+// http://polymer.github.io/LICENSE.txt
+// Code distributed by Google as part of this project is also
+// subject to an additional IP rights grant found at
+// http://polymer.github.io/PATENTS.txt
+
+let assert = require('chai').assert;
+let {Strategy} = require('../../strategizer/strategizer.js');
+let Recipe = require('../recipe/recipe.js');
+let RecipeWalker = require('../recipe/walker.js');
+
+module.exports = class GroupViewConnections extends Strategy {
```

Comment:
I think we should start adding documentation about what individual strategies do in general. I'm a bit lost coming back to this one.

Reply:
totally agree, we should have documentation
this one is for particles that were added based on search tokens, and don't have any bindings between their viewconnections. 
so i'm trying to connect in viewconnections with out view connections from different particles as much as possible.
this is very speculative, but i figured we should start somewhere...